website/docs/r/virtual_machine.html.markdown

```diff
@@ -1600,6 +1600,44 @@ management of such disks would render their configuration unstable.
 
 [tf-vsphere-virtual-disk]: /docs/providers/vsphere/r/virtual_disk.html
 
+## Virtual Machine Reboot
+
+The virtual machine will be rebooted if any of the following parameters changed:
+
+* `extra_config`
+* `guest_id`
+* `alternate_guest_name`
+* `hardware_version`
+* `pci_device_id`
+* `memory` -  When reducing the memory size, or when increasing the memory size and `memory_hot_add_enabled` is set to `false`
```

Comment:
* `memory` 
   When reducing or when increasing the memory size and when `memory_hot_add_enabled` is set to `false`.

Reply:
I think my statement is more correct here as reducing the memory size even if hot add is enabled will cause reboot, but increase the memory size will cause reboot only if hot add is not enabled